### PR TITLE
Header footer

### PR DESCRIPTION
**Pull request by agnieszka-pytel**
Added header and footer

### CODE REVIEW

File: src/app/components/footer/footer.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
```
Comment (ArtemLunt) [inline src/app/components/footer/footer.component.ts]: Pls use OnPush CD strategy everywhere by default.

File: src/app/components/footer/footer.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+})
+export class FooterComponent implements OnInit {
+
+  constructor() { }
```
Comment (ArtemLunt) [inline src/app/components/footer/footer.component.ts]: Get rid of empty code blocks

File: src/app/components/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<header class="c-header fx fx-xcenter">
```
Comment (ArtemLunt) [inline src/app/components/header/header.component.html]: What means all those prefixes?

File: src/app/components/header/header.component.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
+@import 'placeholders';
+@import 'variables';
+.c-header {
+    @extend %primary-theme;
+    font-size: 1.25em;
+    &__link {
```
Comment (ArtemLunt) [inline src/app/components/header/header.component.scss]: Linebreaks between different parts pls

File: src/styles.scss [hunk 5]
```diff
@@ -22,6 +22,7 @@ $bookshelf-app-theme: mat-light-theme($bookshelf-app-primary, $bookshelf-app-acc
 
 /* You can add global styles to this file, and also import other style files */
 
+@import './assets/styles/';
```
Comment (ArtemLunt) [inline src/styles.scss]: Import whole folder or what?

Comment (agnieszka-pytel) [reply src/styles.scss]: Load index file

File: src/app/components/header/header.component.html (hunk 3 above)
Comment (agnieszka-pytel) [reply src/app/components/header/header.component.html]: fx is just flex, I can change it to full name
c-header is a BOM terminology

**Pull request merged**